client/app/visualizations/sunburst/index.js

```diff
@@ -40,7 +43,7 @@ export default function init(ngModule) {
 
     const editTemplate = '<sunburst-sequence-editor></sunburst-sequence-editor>';
     const defaultOptions = {
-      height: 300,
```

Comment:
If we remove this, we need to remove the option from the viz editor. I'm actually fine with it, if it now occupies the size it is given.

(this is the same for Sankey)

Reply:
Need to check (probably for other components too). I think, in a case when user can set height of viz directly on dashboard - we don't need such option in editors, it's redundant.

Reply:
I agree. While I can think of uses for this, it's not worth the extra complexity.